pkg/virt-operator/resource/apply/rbac.go

```diff
@@ -263,22 +264,59 @@ func arePolicyRulesEqual(role1 runtime.Object, role2 runtime.Object) (equal bool
 
 		return true
 	}
+	changeExistingPolicyRulesByRequired := func(existing, required *[]rbacv1.PolicyRule) (changed bool) {
+		if !arePolicyRulesEqual(*existing, *required) {
+			*existing = *required
+			return true
+		}
+		return false
+	}
+	changeExistingSubjectsByRequired := func(existingSubjects, requiredSubjects *[]rbacv1.Subject) (changed bool) {
+		if len(*existingSubjects) != len(*requiredSubjects) {
+			*existingSubjects = *requiredSubjects
+			return true
+		}
+
+		for i := 0; i < len(*existingSubjects); i++ {
```

Comment:
are we guaranteed order?

Reply:
Good point. Thanks.
I will also add a test for trying to make no changes but changing the order of subjects.